src/portal/lib/nodes.ts

```diff
@@ -422,9 +429,15 @@ export async function getDNodes(
   }[] = [];
   for (const node of nodes) {
     const price = countPrice(pricing, node);
-    const [discount, discountLevel] = await calDiscount(api, address, pricing, price);
-    const ips = await getIpsForFarm(node.farmId);
+    const [discount, discountLevel] = await calDiscount(TFTbalance, pricing, price);
+    const ips = "";
     dNodes.push({
+      farm: {
+        id: node.farmId,
+        pubIps: "",
+        name: "",
+        farmCertType: "",
```

Comment:
Why do we publish empty values here?

Reply:
to be accessible later on adding the data for each node